ui/packages/consul-ui/app/components/menu/index.hbs

```diff
@@ -0,0 +1,16 @@
+<ul
+  role="menu"
+  aria-labelledby={{@disclosure.button}}
+  id={{@disclosure.panel}}
```

Comment:
I guess I like that there is no confusion around `panel_id` or was it `panelid`, or did I call it `panelId`, or maybe `panelID`, what about `panel_ID`? 🤔 

So just `panel` works best for me.

Also, I think I documented these props in the Disclosure components docs.

Interested in how you knew it was a `string`?

Reply:
I knew because the `id` attribute expects a string.  But on a cursory read I would have expected the value to be an element or a component, since it is unqualified.

Reply:
Ah gotcha, so all our components should be `CapitalCased`, even `contextual.Components`. We usually use a dollar sign `$element` to denote a DOM element, but we should actually do that more in our templates if we are exporting/yielding `$elements` 🤔 not sure if we ever export/yield DOM elements from components though. But something I'll keep in mind for where I see that we do.